modules/core/src/geometry/geometry.js

```diff
@@ -92,7 +92,6 @@ export default class Geometry {
       if (attributeName === 'indices') {
         assert(!this.indices);
         delete attribute.isIndexed;
-        delete attributes[attributeName];
```

Comment:
Is deleting this going to change the model behavior (creating an extra buffer)? Can we just make a copy of the `attributes` object?

Reply:
No, this was just modifying the input object. Buffers are created based on `this.attributes`.